packages/paste-website/src/pages/components/progress-steps/index.mdx

```diff
@@ -0,0 +1,399 @@
+export const meta = {
```

Comment:
issue: this page is failing to build because we need to list the progress steps package in the website package.json file

Reply:
Thanks!